test-integration/filter-aaveV1.js

```diff
@@ -151,33 +68,20 @@ contract("AaveV1 Filter", (accounts) => {
     });
 
     it("should allow deposits of ERC20 on behalf of wallet", async () => {
-      // Fund the wallet with 1000 USDC tokens
-      const masterMinter = "0xe982615d461dd5cd06575bbea87624fda4e3de17";
-      await usdcToken.configureMinter(accounts[0], web3.utils.toWei("10000"), { from: masterMinter });
-      await usdcToken.mint(wallet.address, 1000);
-      const usdcTokenBalance = await usdcToken.balanceOf(wallet.address);
-      expect(usdcTokenBalance).to.eq.BN(1000);
-
-      const transactions = encodeCalls([
-        [usdcToken, "approve", [aaveLendingPoolCore, 1000]],
-        [aaveLendingPool, "deposit", [usdcToken.address, 1000, ""]]
+      expect(await argent.USDC.balanceOf(wallet.address)).to.gte.BN(1000);
```

Comment:
we could add `.be` to the chain to "sound" better: `expect(...).to.be.gte.BN(1000)`

Reply:
Done